lib/temperature/index.js

```diff
@@ -41,7 +47,7 @@ function TemperatureSensor(options) {
 
   this.options = options || {};
   this.options.frequency = this.options.frequency || 1;
-  this.options.controller = this.options.controller || 'bmp085';
+  this.options.controller = this.options.controller || 'Simulator';
```

Comment:
it should be lowcase here to align others

Reply:
ok

Reply:
So if I change this one to s then it doesn't work anymore. eslint complained about s instead of S when creating the object. I'll go back to s everywhere and ignore eslint.

Reply:
yes for the var name:
```
var Simualor = requiere('simulator')
```

param is in lowercase in options like others